swift-sdk/IterableEmbeddedMessagingUpdateDelegate.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  Copyright © 2023 Iterable. All rights reserved.
+//
+
+import Foundation
+
+@objc public protocol IterableEmbeddedMessagingUpdateDelegate {
+    func onMessagesUpdated()
+}
```

Comment:
assuming this is added on another PR

Reply:
Yeah, I had started on implementing update delegates, but then I saw a need to implement the processor first.